file-formats/sajc/type/zif_aff_sajc_v1.intf.abap

```diff
@@ -17,11 +17,15 @@ INTERFACE zif_aff_sajc_v1
       "! ABAP language version
       "! $values {@link zif_aff_types_v1.data:co_abap_language_version_cloud}
       "! $default {@link zif_aff_types_v1.data:co_abap_language_version_cloud.standard}
-      abap_language_version TYPE zif_aff_types_v1=>ty_abap_language_version,
+      abap_language_version TYPE zif_aff_types_v1=>ty_abap_language_version_cloud,
     END OF ty_header.
 
+  "! <p class="shorttext">Report Name</p>
+  "! Name of the report
+  TYPES ty_report_name TYPE c LENGTH 40.
+
   TYPES:
-    "! <p class="shorttext">Application Job Catalog Entry</p>
+    "! <p class="shorttext">General Information</p>
```

Comment:
I think we talked about this before a while: We do not want object typer owner to have "General Information" in their AFF type/schema. When they want to have it as section title in their editor, they have to add it to their configuration. So I would update this here.